### ISSUE TITLE
Cannot read property 'some' of undefined: TypeError: Cannot read property 'some' of undefined 	at linterDisabled (C:\Users\Aurelien Pupier\.vscode\extensions\ms-kubernetes-tools.vscode-kubernetes-tools-0.1.15\out\src\extension.js:1740:28)

### ISSUE THREAD

**Issue by apupier** [Azure/vscode-kubernetes-tools#449]
in output -> Log (Window) I have this error:

```
[2019-01-23 08:37:23.592] [renderer1] [error] Cannot read property 'some' of undefined: TypeError: Cannot read property 'some' of undefined
	at linterDisabled (C:\Users\Aurelien Pupier\.vscode\extensions\ms-kubernetes-tools.vscode-kubernetes-tools-0.1.15\out\src\extension.js:1740:28)
	at linters_1.linters.filter (C:\Users\Aurelien Pupier\.vscode\extensions\ms-kubernetes-tools.vscode-kubernetes-tools-0.1.15\out\src\extension.js:1753:29)
	at Array.filter (<anonymous>)
	at C:\Users\Aurelien Pupier\.vscode\extensions\ms-kubernetes-tools.vscode-kubernetes-tools-0.1.15\out\src\extension.js:1753:14
	at Generator.next (<anonymous>)
	at __awaiter (C:\Users\Aurelien Pupier\.vscode\extensions\ms-kubernetes-tools.vscode-kubernetes-tools-0.1.15\out\src\extension.js:7:71)
	at new Promise (<anonymous>)
	at __awaiter (C:\Users\Aurelien Pupier\.vscode\extensions\ms-kubernetes-tools.vscode-kubernetes-tools-0.1.15\out\src\extension.js:3:12)
	at kubernetesLint (C:\Users\Aurelien Pupier\.vscode\extensions\ms-kubernetes-tools.vscode-kubernetes-tools-0.1.15\out\src\extension.js:1743:12)
	at Array.forEach (<anonymous>)
	at C:\Users\Aurelien Pupier\.vscode\extensions\ms-kubernetes-tools.vscode-kubernetes-tools-0.1.15\out\src\extension.js:298:40
	at Generator.next (<anonymous>)
	at fulfilled (C:\Users\Aurelien Pupier\.vscode\extensions\ms-kubernetes-tools.vscode-kubernetes-tools-0.1.15\out\src\extension.js:4:58)
	at <anonymous>
```

**Comment by itowlson** [comment]
PR already out (#448).  Thanks for reporting though, and apologies for letting this through!

**Comment by itowlson** [comment]
Closed via #448.
